GitCommands/StringExtensions.cs

```diff
@@ -189,6 +190,20 @@ public static string Quote(this string? s, string q = "\"")
             return string.IsNullOrEmpty(s) ? s : s.Quote();
         }
 
+        /// <summary>
+        /// Quotes this string for use as a command line argument.
```

Comment:
```suggestion
        /// Quotes the supplied string for use as a command line argument.
```

Reply:
This would mismatch with all the other `Quote` method documentations. Also, this is an extension method, so `this string` makes sense.